plugin/rewrite/name.go

```diff
@@ -58,62 +114,62 @@ const (
 
 // Rewrite rewrites the current request based upon exact match of the name
 // in the question section of the request.
```

Comment:
this comment is not true anymore, becasuse the rule is returned instead of executed?